src/Config.php

```diff
@@ -36,8 +36,13 @@ public static function instance(): self
         }
 
         $basePath = dirname(array_keys(ClassLoader::getRegisteredLoaders())[0]);
+        $configFile = "{$basePath}/peck.json";
 
-        $contents = (string) file_get_contents($basePath.'/peck.json');
+        if ( !file_exists($configFile) ) {
+            $configFile = dirname(__DIR__) . '/peck.json';
```

Comment:
Does this take the projects src/ folder or the root directory? 

If it takes the file from the root directory and the CI passes, everything looks good!

Reply:
First, It looks for the **peck.json** in the root (app base dir), if not exists, It will load the src peck.json file.
would be better to use it as defaut config file i think

Reply:
Wouldn’t it be better if it merged the two files?
Otherwise, you would have to reconfigure everything every time.
But if the files are merged, Peck could come with a preset of words.
I just tested Peck in a small Laravel project and had to define 60 basic PHP terms as ignored.
Nobody wants to do that in every project over and over again.

Reply:
> Wouldn’t it be better if it merged the two files? 

Got your idea 'merged the two files', but you should only keep project "peck" config file in the repo, The maintainer could go further by creating an initial "peck.json" file during composer install (as instance if not exists).

Peck is under alpha version! We are still working on it with Nuno and the contributors. (you can follow him for his streams about peck).